packages/react-app/src/components/pages/Bounties/Filters/index.tsx

```diff
@@ -258,15 +270,37 @@ const MinMaxFilter = ({ name, filters, setFilters }: {
 
 	return (
 		<>
-			{name && <Heading size="xs">{name}</Heading>}
 			<HStack my="2">
-				<Input placeholder="Min" onChange={updateMin} />
-				<Input placeholder="Max" onChange={updateMax} />
+				<Input placeholder="Min Bounty Value" onChange={updateMin} />
+				<Input placeholder="Max Bounty Value" onChange={updateMax} />
 			</HStack>
 		</>
 	);
 };
 
+const SaveFilter = ({ filters }: {filters: FilterParams}): JSX.Element => {
```

Comment:
I like this functionality. 

Though - it takes up a good amount of real estate. 100% does not need to be addressed this CR, but am wondering if there's other functionality we can prioritize to grab the user's attention instead